### ISSUE TITLE
Can we change storage.volume to just plain storage in importable attributes?

### ISSUE THREAD

**Issue by grdw** [quintel/etmoses#1012]
It's causing a bit of trouble trying to fetch specific attributes from et-engine with plain technologies. Since ETEngine whitelists 'storage' but not 'storage.volume'.

**Comment by antw** [comment]
`storage` should be a hash containing a `volume` key and, optionally, a `decay` key. Doesn't ETE just return the whole hash and then `Import::Attribute` extracts the volume?

**Comment by grdw** [comment]
> Doesn't ETE just return the whole hash 

Not really... it returns a: `#<Atlas::StorageDetails:0x007fc05e39bce8>` object.

**Comment by antw** [comment]
Could you clarify what "a bit of trouble" means though? ETE has always happily returned the `storage` attribute in the past; what has happened to make that more difficult?

**Comment by grdw** [comment]
> ETE has always happily returned the storage attribute in the past; what has happened to make that more difficult?

To cearify. I just recently started to whitelist the attributes list for the technologies. Where it previously was the case where we would just ask ETEngine to return a statically defined list of attributes, that is now dynamic. So in order to also do this for the default list of technologies I stumbled upon the `storage` issue. 

I think the reason why `storage.volume` worked great is because it's not whitelisted but you ask for the `storage` attribute.

I need to fiddle with it some more.

**Comment by grdw** [comment]
After talking with @ChaelKruip about this issue. It's pretty clear that this is the hurdle that will probably cause a lot of other hurdles. 

I'll change the name + description of this issue and will fix it when time is due.
